src/diffcalc_api/routes/ub.py

```diff
@@ -1,13 +1,12 @@
 """Endpoints relating to the management of setting up the UB calculation."""
 
-from typing import List, Optional
+from typing import List, Optional, Union
 
 from fastapi import APIRouter, Body, Depends, Query
 
-from diffcalc_api.config import VECTOR_PROPERTIES
-from diffcalc_api.errors.ub import (
+# from diffcalc_api.config import VECTOR_PROPERTIES
+from diffcalc_api.errors.ub import (  # InvalidPropertyError,
```

Comment:
Should this commented out code be deleted?

Reply:
Yep, deleted this and the thing it's trying to import as this was its only purpose